service/geopmdpy/__init__.py

```diff
@@ -34,6 +34,7 @@
 
 """
 
+from __future__ import absolute_import
```

Comment:
This is only for supporting python 2.x, we really want to get rid of all of these abolute_import lines, not add them back

Reply:
Should I do this a part of this PR, or a new one?